src/query.js

```diff
@@ -54,7 +54,7 @@ const cleanSettings = stateObj => {
   return nextState;
 };
 
-export function writeHash(mapSettings) {
+export function getHashString(mapSettings) {
```

Comment:
What do you think of renaming this to something like `makeHashString` or `createHashString`? I feel like `getHashString` makes me think we're retrieving a part of `mapSettings` or retrieving something from the actual hash.